doc/man1/genrsa.pod

```diff
@@ -112,6 +121,15 @@ Because key generation is a random process the time taken to generate a key
 may vary somewhat. But in general, more primes lead to less generation time
 of a key.
 
+For multi-prime RSA, which is defined in RFC 8017, unlike normal 2-prime RSA,
+its security depends on both how many primes are used and how long the RSA
+modulus is. But it's still estimated by the difficulty of factoring the modulus.
+In simple terms, the more primes a multi-prime RSA key has, the easier it is for
```

Comment:
I don't like starting a sentence with "But" and no comma; maybe "The security is still related to the difficulty of factoring the modulus -- in simple terms, [...]"